test/random.jl

```diff
@@ -537,3 +537,23 @@ let g = Base.Random.GLOBAL_RNG,
     @test srand(m, rand(UInt32, rand(1:10))) === m
     @test srand(m, rand(1:10)) === m
 end
+
+# test randstring API
+let b = ['0':'9';'A':'Z';'a':'z']
+    for rng = [[], [MersenneTwister(0)]]
+        @test length(randstring(rng...)) == 8
+        @test length(randstring(rng..., 20)) == 20
+        @test issubset(randstring(rng...), b)
+        for c = ['a':'z', "qwèrtï", Set(Vector{UInt8}("gcat"))],
+            len = [8, 20]
```

Comment:
I'd indent this one level further or include it on the same line as the rest of the for, it looks like an assignment rather than a loop variable if you miss the final comma on the previous line